src/Microsoft.AspNetCore.Server.Kestrel/Networking/PlatformApis.cs

```diff
@@ -21,41 +21,13 @@ static PlatformApis()
             if (!IsWindows)
             {
                 // When running on Mono in Darwin OSVersion doesn't return Darwin. It returns Unix instead.
```

Comment:
The comment is still valid. Though we could replace all of this code with platform abstractions.